scout/server/blueprints/variants/views.py

```diff
@@ -533,8 +533,12 @@ def upload_panel(institute_id, case_name):
 @variants_bp.route("/verified", methods=["GET"])
 def download_verified():
     """Download all verified variants for user's cases"""
-    user_obj = store.user(current_user.email)
-    user_institutes = user_obj.get("institutes")
+
+    user_institutes = (
+        [inst["_id"] for inst in store.institutes()]
+        if current_user.is_admin
+        else current_user.get("institutes")
```

Comment:
Oh, but that won't work for non admins, right? LoginUser is a UserMixin, not a Scout user obj/dict?
```
2022-01-27 13:04:58 k1315C02VR0WRHTDH scout.server.app[88833] ERROR Exception on /verified [GET]
Traceback (most recent call last):
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/dannil/sandbox/scout/scout/server/blueprints/variants/views.py", line 540, in download_verified
    else current_user.get("institutes")
  File "/usr/local/miniconda3/envs/scout39/lib/python3.9/site-packages/werkzeug/local.py", line 347, in __getattr__
    return getattr(self._get_current_object(), name)
AttributeError: 'LoginUser' object has no attribute 'get'
```

Reply:
```suggestion
        else user_obj.get("institutes")
```